### PR TITLE
Refactor approvers

### PR DESCRIPTION
**Pull request by fbogsany**
This adds @robertlaurin to the list of Approvers for OpenTelemetry Ruby. Robert contributed the Resource auto-detectors. He also maintains Shopify's wrapper gem for OpenTelemetry.

This also proposes removing @bai, @luvtechno and @elskwid from the Approvers list. They have not been active in this project for many months, at least. We're very grateful for @elskwid's early contributions, in particular, however Approver status requires continued active involvement in the project.

### CODE REVIEW

**Comment by fbogsany** [comment]
I’ll keep this open for 48 hours, in case anyone wants to voice concerns or objections.

**Review by bai — APPROVED** [review]

**Comment by fbogsany** [comment]
I've updated this to add @ericmustin as an Approver as well, after discussion with @mwear. This is blocked on https://github.com/open-telemetry/community/issues/460.

**Pull request merged**
